MvvmCross/Platforms/Android/Views/MvxApplicationCallbacksCurrentTopActivity.cs

```diff
@@ -10,7 +10,7 @@ namespace MvvmCross.Platforms.Android.Views
 {
     public class MvxApplicationCallbacksCurrentTopActivity : Java.Lang.Object, Application.IActivityLifecycleCallbacks, IMvxAndroidCurrentTopActivity
     {
-        private ConcurrentDictionary<string, ActivityInfo> _Activities = new ConcurrentDictionary<string, ActivityInfo>();
+        protected ConcurrentDictionary<string, ActivityInfo> _Activities { private set; } = new ConcurrentDictionary<string, ActivityInfo>();
```

Comment:
Shouldn't it have a `get` too?

Reply:
Yep, it's missing that.

Reply:
I've updated it, together with the ActivityInfo visibility.

Reply:
I'd also drop the `_` from the protected member name